examples/flagging-bad-clients/README.md

```diff
@@ -14,10 +14,13 @@ To each redirected message we add user properties to identify bad clients and to
 - [Docker compose](https://docs.docker.com/compose/) 
 - A HiveMQ license with enabled Data Governance Hub (contact [datagovernancehub@hivemq.com](mailto:datagovernancehub@hivemq.com))
 - A [HiveMQ Enterprise Extension for PostgreSQL](https://www.hivemq.com/extension/postgresql-extension/) license. If no license is provided, a trial license is valid for 5 hours.
-Please add your license according the [Dockerfile](hivemq/Dockerfile) 
+Please add your license according the [Dockerfile](hivemq/Dockerfile)-
+- (**License**): If you have a HiveMQ license with Data Governance Hub copy the file into the container `hivemq/hivemq.plic` (see `docker-compose.yml` commented line). 
+- (**Non-License**): In case you don't have a license, the trial mode is activated which stays active for 5 hours.
```

Comment:
Shouldn't this be an optional point in quickstart? 
The license is already mentioned two points above.

Also the formatting seems off:
```md
- Please add your license according the [Dockerfile](hivemq/Dockerfile):
  - (**License**): If you have a HiveMQ license with Data Governance Hub copy the file into the container `hivemq/hivemq.plic` (see `docker-compose.yml` commented line). 
  - (**Non-License**): In case you don't have a license, the trial mode is activated which stays active for 5 hours.
```